lib/hammer_cli_foreman/compute_resource/libvirt.rb

```diff
@@ -2,48 +2,48 @@ module HammerCLIForeman
   module ComputeResources
     class Libvirt < Base
       def name
-        _('Libvirt')
+        'Libvirt'
       end
 
       def compute_attributes
         [
-            ['cpus',   _('Number of CPUs')],
-            ['memory', _('String, amount of memory, value in bytes')],
-            ['start',  _('Boolean (expressed as 0 or 1), whether to start the machine or not')]
+          ['cpus',   _('Number of CPUs')],
+          ['memory', _('String, amount of memory, value in bytes')],
+          ['start',  _('Boolean (expressed as 0 or 1), whether to start the machine or not')]
```

Comment:
this line is not necessary, you already have 'start' in host_attributes

Reply:
Since it was there before this PR, I missed this one, thanks :)